### PR TITLE
Adding SGE 

### PR DESCRIPTION
**Pull request by reeves89**
Let me know if I need to add anything.

### CODE REVIEW

**Comment by HeesungB** [comment]
@reeves89   Hello. 
Can you edit the file using the link below?
https://github.com/chainapsis/keplr-chain-registry#requirement-details

You can also use the command below to check the validation locally.
```
yarn validate cosmos/sgenet.json
```

**Comment by reeves89** [comment]
Hello, @HeesungB Thanks for your revert. Updated, Let me know if you need any thing.

File: cosmos/sgenet.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      ],
+      "stakeCurrency": {
+        "coinDenom": "SGE",
+        "coinMinimalDenom": "usge",
+        "coinDecimals": 6,
+        "coinGeckoId": "six-sgma"
+      },
+      "features": [
+        "cosmwasm",
+        "ibc-transfer"
```
Comment (HeesungB) [inline cosmos/sgenet.json:51]: @reeves89 Could you remove `"ibc-transfer"` on features?

**Comment by HeesungB** [comment]
@reeves89 
Could You resize image to 256x256px size?
and Could you remove `coingekcoId` field? and You can add after token price shown.

**Comment by reeves89** [comment]
> @reeves89 Could You resize image to 256x256px size? and Could you remove `coingekcoId` field? and You can add after token price shown.

@HeesungB updated.

**Pull request merged**